### PR TITLE
Added 1.0

### PR DESCRIPTION
**Pull request by Crabbey**
Added version 1.0 :)

### CODE REVIEW

**Comment by EVOTk** [comment]
Big Thx :D

**Comment by Fank** [comment]
Thanks for the PR the tags should be pushed now

File: 1.0/files/config.ini [hunk 1]
```diff
@@ -0,0 +1,755 @@
+; version=8
```
Comment (SuperSandro2000) [inline 1.0/files/config.ini:1]: Did you update this file? Probably not, so could you please do that? @Crabbey

Comment (Fank) [reply 1.0/files/config.ini:1]: Did something changed in there? I didn't noticed anything.

Comment (SuperSandro2000) [reply 1.0/files/config.ini:1]: I am not sure if something did. Thats why we need to check it.

**Pull request merged**